bedrock/products/templates/products/vpn/more/vpn-or-proxy.html

```diff
@@ -0,0 +1,44 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "products/vpn/more/base-article.html" %}
+
+{% block page_title %}{{ ftl('vpn-or-proxy-the-difference-between') }}{% endblock %}
+{% block page_desc %}{{ ftl('vpn-or-proxy-vpns-and-proxies-are') }}{% endblock %}
+
+{% block article %}
+<p class="mzp-c-article-intro">{{ ftl('vpn-or-proxy-virtual-private-networks') }}</p>
+
+<h3>{{ ftl('vpn-or-proxy-stop-isps-from') }}</h3>
+<p>{{ ftl('vpn-or-proxy-when-you-use-firefox', firefox=url('firefox.new'), etp='https://blog.mozilla.org/products/firefox/control-trackers-with-firefox/', trackers='https://blog.mozilla.org/internet-culture/mozilla-explains/what-is-a-web-tracker/') }}</p>
+<p>{{ ftl('vpn-or-proxy-because-your-internet') }}</p>
+<p>{{ ftl('vpn-or-proxy-isps-can-use') }}</p>
+
+<h3>{{ ftl('vpn-or-proxy-when-should-you') }}</h3>
+<p>{{ ftl('vpn-or-proxy-vpns-and-secure') }}</p>
+
+<figure class="c-article-figure">
+  {{ high_res_img('img/products/vpn/more/vpn-proxy.png', {'alt': '', 'width': '960', 'height': '866'}) }}
```

Comment:
Not entirely sure how you might handle this, but this image feels like it's more than decorative and could maybe use some `alt` text?

![image](https://user-images.githubusercontent.com/400117/127290947-aa3bfc88-6fd8-46ba-8a2d-7166be1e072f.png)

The difference is explained in the paragrahps below, but the image does serve as a nice quick summary.

Reply:
I thought about it, but since the text does explain everything I decided the illustration was supplemental. Trying to describe it adequately in a text alternative would probably have to repeat much of the same info in a condensed summary, and not actually describe the image. So I think it's probably OK to leave alt blank and treat this image as optional.